### ISSUE TITLE
[NEXT] Cannot import `useLocalePath` manually. TypeError: Cannot read properties of undefined (reading 'Symbol(vue-i18n-routing-gor)')

### ISSUE THREAD

**Issue by szulcus** [nuxt-modules/i18n#1596]
### Version
@nuxtjs/i18n: 8.0.0-beta.2
nuxt: 3.0.0-rc.12

### Nuxt configuration

Please change to `[x]` if relevant for this issue:

 - [x ] Applies to a site deployed to a static server (site generated with `nuxt generate`)
 - [ ] Applies to a site deployed to a server with a Node backend

### Reproduction Link
https://stackblitz.com/edit/github-1pmwyi?file=nuxt.config.ts
The reproduction shows an additional error: "Not available in legacy mode" 🤷‍♂️

### Steps to reproduce
In .vue file
```ts
import { useLocalePath } from 'vue-i18n-routing';

const localePath = useLocalePath();
console.log(localePath('index'));
```

### What is Expected?
Working localePath.

### What is actually happening?
`localePath` work only if is auto-imported.
```
Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'Symbol(vue-i18n-routing-gor)')
    at getGlobalOptions (vue-i18n-routing.mjs:457:16)
    at getI18nRoutingOptions (vue-i18n-routing.mjs:586:19)
    at Object.resolveRoute (vue-i18n-routing.mjs:633:130)
    at Object.localePath (vue-i18n-routing.mjs:618:39)
    at vue-i18n-routing.mjs:895:20
    at setup (index.vue:8:2)
    at callWithErrorHandling (runtime-core.esm-bundler.js:155:22)
    at setupStatefulComponent (runtime-core.esm-bundler.js:7204:29)
    at setupComponent (runtime-core.esm-bundler.js:7159:11)
    at mountComponent (runtime-core.esm-bundler.js:5508:13)
```

**Comment by kazupon** [comment]
>The reproduction shows an additional error: "Not available in legacy mode" 🤷‍♂️

This is an issue, so I'll fix it.

>import { useLocalePath } from 'vue-i18n-routing';

BTW, In @nuxtjs/i18n, `useLocalePath` is exported from `vue-i18n-routing` is doesn't work (and other composable functions)
So, You need to use auto-imported compsable functions.

**Comment by szulcus** [comment]
@kazupon it's a pity i have used it so far 😥. I think users should be able to explicitly import and know where composables is imported from. Sometimes there are so many composables that it's hard to remember everything. Not to mention the new people to the project. But it's your decision.

If someone like me has auto-imports disabled, you can use this workaround:

```ts
import { useLocalePath } from '#imports';
```

**Comment by kazupon** [comment]
@szulcus 
You can use import with  `#imports` 
```ts
import { useI18n, useLocalePath } from '#imports'
```

**Comment by kazupon** [comment]
I've just released v8.0.0-beta.3
close
